.github/workflows/publish.yml

```diff
@@ -26,4 +26,4 @@ jobs:
       - name: Publish
         run: npm publish
```

Comment:
I'm not sure but I guess this is going to fail, anyway we can merge it and fix it in due time, It's more important to have the main pipeline